allure-framework-modules/allure-webapi-module/src/main/java/com/capgemini/ntc/example/core/base/runtime/RuntimeParameters.java

```diff
@@ -0,0 +1,74 @@
+package com.capgemini.ntc.example.core.base.runtime;
+
+import com.capgemini.ntc.test.core.base.runtime.RuntimeParametersI;
+import com.capgemini.ntc.test.core.logger.BFLogger;
+
+/**
+ * This class stores various system properties These parameters are accessible while test case executes Example :
+ * PARAM_3("param_3", "1410") IS: mvn test -Dparam_3=1525 -Dtest=MyTestClass
+ * 
+ * @author LUSTEFAN
+ */
+public enum RuntimeParameters implements RuntimeParametersI {
```

Comment:
Overall question,  what runtime parameters you would need to have in WebAPI startup. 
These parameters are taken while you run maven 
`mvn test -Dparam_3=1525`

Reply:
Under consideration in future